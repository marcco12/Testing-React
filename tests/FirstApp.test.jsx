
import { render } from "@testing-library/react"
import { FirstApp } from "../src/FirstApp"

describe('Pruebas en <FirstApp />', () => { 
    test('should hacer macth con el snapshot', () => { 
        const title = 'Soy Vegeta';
        const {container} = render( <FirstApp title={title}/> )
        expect(container).toMatchSnapshot();
     });

     test('should mostrar el tÃ­tulo en un H1', () => { 
        const title = 'Soy Vegeta';
        const {container, getByText} = render( <FirstApp title={title}/> )
        
        expect(getByText(title)).toBeTruthy();
      })
 })