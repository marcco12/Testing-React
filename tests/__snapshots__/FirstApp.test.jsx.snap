// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <FirstApp /> should hacer macth con el snapshot 1`] = `
<div>
  <h1
    data-testid="test-title"
  >
     
    Soy Vegeta
     
  </h1>
  <p>
    No hay subtítulo
  </p>
  <p>
    No hay subtítulo
  </p>
  <p>
    Fernando Herrera
  </p>
</div>
`;
